python/py-pytest-metadata/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+set real_name       pytest-metadata
+name                py-${real_name}
+version             1.8.0
+revision            0
+categories-append   devel
+
+license             Mozilla-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Plugin for accessing test session metadata
+
+long_description    pytest-metadata is a plugin for pytest that\
+                    provides access to test session metadata.
+
+homepage            https://pypi.python.org/pypi/${real_name}
+
+master_sites        pypi:p/${real_name}/
+distname            ${real_name}-${version}
+
+checksums           rmd160  722a1d9d4121fd5314e3eeda69e678198af06ae6 \
+                    sha256  2071a59285de40d7541fde1eb9f1ddea1c9db165882df82781367471238b66ba \
+                    size    9227
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools_scm
+
+    depends_run-append \
+                    port:py${python.version}-pytest
+
+    depends_test-append \
+                    port:py${python.version}-pytest
+
+    post-destroot {
+        set docdir ${prefix}/share/doc/${subport}
+        xinstall -d ${destroot}${docdir}
+        xinstall -m 0644 -W ${worksrcpath} README.rst LICENSE \
+            ${destroot}${docdir}
+    }
+
+    test.run        yes
+    test.env        PYTHONPATH=${worksrcpath}/build/lib
```

Comment:
this doesn't work, there are no tests run with ```python setup.py test```. You say in the PR description that you tested this, ideally you would have noticed this. Instead replace these two lines with:
```
    test.run        yes
    test.cmd        py.test-${python.branch}
    test.target
    test.env        PYTHONPATH=${worksrcpath}/build/lib
```

Reply:
The tests subdir exists, https://github.com/pytest-dev/pytest-metadata/tree/master/tests

I don't see the issue.

Reply:
same comment as I gave earlier - if you would actually run ```sudo port -dv test py37-pytest-metadata```, you should see as the output: ```Ran 0 tests in 0.000s```. How is that not an issue?

You are assuming that you can run the tests with ```python3.7 setup.py --no-user-cfg test```, which is apparently not the case. Hence I gave you the correct way of setting up the test phase; the idea would be that you try that and learn from it.

Reply:
Thanks done. 🦃🦃🦃